go.mod

```diff
@@ -45,8 +46,10 @@ require (
 	github.com/urfave/negroni v0.3.0
 	go.etcd.io/etcd v0.5.0-alpha.5.0.20191023171146-3cf2f69b5738
 	go.uber.org/goleak v0.10.0
-	go.uber.org/zap v1.13.0
+	go.uber.org/zap v1.15.0
 	golang.org/x/tools v0.0.0-20200527183253-8e7acdbce89d
 	google.golang.org/grpc v1.25.1
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 )
+
+replace github.com/pingcap/errors => github.com/YuJuncen/errors v0.8.1-0.20200727030200-a512ae64de43
```

Comment:
Does the branch already merge to master?

Reply:
No, after that PR is merged, we can update the latest master